elasticsearch/examples/hot-cold/Chart.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+description: Elasticsearch Hot Cold Example
+home: https://github.com/elastic/helm-charts
+name: elasticsearch-host-cold
```

NIT to make a bit more explicit that it's an example and not a real chart supported by Elastic.
```suggestion
name: elasticsearch-host-cold-example
```